packages/client/src/client.ts

```diff
@@ -443,7 +443,7 @@ export class StandardLogtoClient {
     }
 
     const accessTokenKey = buildAccessTokenKey(resource, organizationId);
-    const { appId: clientId } = this.logtoConfig;
+    const { appId: clientId, scopes } = this.logtoConfig;
```

Comment:
Hi, I thought the plan was to add a "scope" parameter to the function `getAccessTokenByRefreshToken`. Reading `scopes` from `logtoConfig` does not really make sense here, because the `scopes` were already sent in the auth request with exactly the same value.

Reply:
Hey @wangsijie,

The issue for me was that I handle the sign in, etc and then call `getAccessToken()`, which returns an opaque access token. I follow that up with a call to `getAccessToken(resourceUrl)` in order to get a JWT with aud set to the resourceUrl. However, at the moment, that accessToken has no scopes, because they're not passed from the client's options. When I did the initial setup, before diving into the code, I assumed they would be because I had set them when I initialised the client.

The alternative would be to add the scopes as a third argument to `getAccessToken`

Reply:
Hi @maddijoyce, I just did a test, init Logto client with resource (`https://my-resource.com/api`) and scopes (`profile`, `test`), and then call `getAccessToken('https://my-resource.com/api')`, the JWT contains the scope `test` which is expected. Can you please try again?

Reply:
I can confirm that your configuration is correct and the user should be getting this scope. The issue might be that the user doesn’t have `test_scope` access on this resource, could you double check your RBAC configuration?